doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -182,8 +182,8 @@ void qsort(void *base, size_t nmemb, size_t size,
 The `base` argument is a pointer to an array of length `nmemb`, with elements of `size` bytes
 each. `compare` is a callback function which takes pointers to two elements `a` and `b` and returns
 an integer less/greater than zero if `a` should appear before/after `b` (or zero if any order
-is permitted). Now, suppose that we have a 1d array `A` of values in Julia that we want to sort
-using the `qsort` function (rather than Julia's built-in `sort` function). Before we worry about
+is permitted). Now, suppose that we have a 1-d array `A` of values in Julia that we want to sort
```

Comment:
`1d` is acceptable spelling here and is more consistent with the spelling elsewhere in the manual.

Reply:
I changed it to 1-d because that's what I've seen elsewhere in the docs.